content/spin/api-guides-overview.md

```diff
@@ -0,0 +1,24 @@
+title = "API Guides Overview"
+template = "spin_main"
+date = "2023-03-03T03:03:03Z"
+
+---
+
+### Spin's Hosting Capabilities and Interfaces
```

Comment:
"Hosting capabilities" implies its abilities to host things.  "Hosted" maybe?  I am not sure of a good phrase here (which is why I went for APIs).

Reply:
it just occurred to me... do we even need a heading here...?  I think Bartholomew already displays the page title as a big heading before the first para anyway...?

Reply:
That is a good point; two headings in a row is unnecessary anyway.